diesel_migrations/migrations_internals/src/migration.rs

```diff
@@ -54,9 +54,15 @@ impl<'a> fmt::Display for MigrationFileName<'a> {
     }
 }
 
-pub fn migration_from(path: PathBuf) -> Result<Box<dyn Migration>, MigrationError> {
+pub fn migration_from<Conn>(
+    _conn: &Conn,
```

Comment:
```suggestion
    conn: &Conn,
```

Reply:
No, because `conn` is only used in barrel, so if barrel is not enable, your change will give warning

Reply:
Oh - my bad (I didn't see the feature flag)!